x-pack/test/functional/apps/discover/reporting.ts

```diff
@@ -95,20 +60,13 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(res.get('content-type')).to.equal('text/csv; charset=utf-8');
 
         const csvFile = res.text;
-        const lines = csvFile.trim().split('\n');
-
-        // verifies the beginning and end of the text
-        expectSnapshot(lines.slice(0, 100)).toMatch();
-        expectSnapshot(lines.slice(-100)).toMatch();
-
-        expectSnapshot(csvFile.length).toMatchInline(`5093456`);
-        expectSnapshot(lines.length).toMatchInline(`32726`);
+        expectSnapshot(csvFile).toMatch();
       });
 
       it('generates a report from a new search with data: discover:searchFieldsFromSource', async () => {
         await setFieldsFromSource(true);
         await PageObjects.discover.clickNewSearchButton();
-        await PageObjects.reporting.setTimepickerInDataRange();
+        await PageObjects.reporting.setTimepickerInEcommerceDataRange();
```

Comment:
Just after this line it might be useful to add an assertion against the number of hits returned by the search:

```
expect(await PageObjects.discover.getHitCount()).to.equal('4,675');
```

That way, if this fails in future, we will know that at least the Discover search is probably working as expected.